rust/relay/server/src/main.rs

```diff
@@ -642,6 +643,21 @@ where
                 ready = true;
             }
 
+            if let Some(ebpf) = self.ebpf.as_mut() {
+                let is_enabled = ebpf.config().relaying_enabled;
+                let should_be_enabled =
+                    firezone_telemetry::feature_flags::ebpf_turn_router_enabled();
+
+                if is_enabled != should_be_enabled {
+                    tracing::info!(%is_enabled, %should_be_enabled, "eBPF router feature-flag changed");
```

Comment:
I believe there is a bug here causing this to be logged on each packet

![Uploading Screenshot 2025-04-03 at 9.57.27 PM.png…]()

Reply:
oops.

https://github.com/firezone/firezone/pull/8655